mangaki/mangaki/tests/test_anidb.py

```diff
@@ -4,22 +4,27 @@
 from django.conf import settings
 from django.test import TestCase
 
-from mangaki.models import Editor, Studio
-from mangaki.utils.anidb import client, AniDB
+from mangaki.models import Category, Editor, Studio, Work, Role, Staff, Artist
+from mangaki.utils.anidb import AniDB
 
 
 class AniDBTest(TestCase):
     @staticmethod
+    def create_anime(**kwargs):
+        anime = Category.objects.get(slug='anime')
+        return Work.objects.create(category=anime, **kwargs)
+
+    @staticmethod
     def read_fixture(filename):
-        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r') as f:
+        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r', encoding='utf-8') as f:
             return f.read()
 
     def setUp(self):
         # FIXME: The defaults for editor and studio in Work requires those to
         # exist, or else foreign key constraints fail.
         Editor.objects.create(pk=1)
-        Studio.objects.create(pk=1)
-        self.anidb = client
+        # Studio.objects.create(pk=1)
```

Comment:
It's pretty dangerous, currently, the `Work` model uses a `default` of `Studio(id=1)` when no studios are specified.
But I guess it's okay since AniDB give us `studios` :D.